Extension/package.json

```diff
@@ -42,6 +42,168 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "walkthroughs": [
+      {
+        "id": "cppWelcome",
+        "title": "Get Started with C++ Development",
+        "description": "Your first steps to set up powerful C++ tools in a lightweight, performant editor!",
+        "steps": [
+          {
+            "id": "verify.compiler.mac",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your computer to configure IntelliSense for your project. [Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
```

Comment:
"C++ extension" should be "C/C++ extension".

Reply:
Copying reply to your previous comment about this:

In blog posts and conference talks we just say C++, and the audience seems to understand that it encompasses both C++ and C. I want to be consistent with our recent branding of calling it "the C++ extension," but I think this would be a good thing to experiment with!